src/components/UserAccess/UserAccessForm/UsernameSection.tsx

```diff
@@ -68,9 +68,9 @@ export const UsernameSection: React.FC<React.PropsWithChildren<Props>> = ({ disa
         label="Enter usernames"
         labelIcon={
           <HelpPopover
-            aria-label="Usernames in RHTAP"
-            headerContent="Usernames in RHTAP"
-            bodyContent="To find their username, the user you want to invite should log into RHTAP and click Workspaces. Their username is the first item in the breadcrumb navigation."
+            aria-label="Usernames in KONFLUX"
```

Comment:
change at other places also
```suggestion
            aria-label="Usernames in Konflux"
```